test/VersionInfo/VersionInfoTests.cpp

```diff
@@ -30,12 +30,6 @@ namespace Test::VersionInfo
 
         TEST_METHOD(VersionInfo_Release)
         {
-            if (::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::Feature_VersionInfoAPI::IsEnabled())
-            {
-                WEX::Logging::Log::Comment(WEX::Common::String(L"Feature_VersionInfoAPI is disabled. Skipping..."));
-                return;
-            }
-
             try
             {
 #if defined(WINDOWSAPPRUNTIME_MICROSOFT_WINDOWS_APPLICATIONMODEL_WINDOWSAPPRUNTIME_FEATURE_VERSIONINFOAPI_ENABLED)
```

Comment:
I believe these #define checks need to be removed as well.

Reply:
right, thx